dev/resources/schema.setup.cql

```diff
@@ -29,17 +29,16 @@ AND  COMPACTION = { 'class' :  'LeveledCompactionStrategy' };
 
 
 CREATE TABLE IF NOT EXISTS lcmap_landsat_dev.tile_specs (
-    keyspace_name text,
-    table_name text,
-    projection text static,
-    tile_x int static,
-    tile_y int static,
+    name text,
```

Comment:
Following yesterday afternoon's discussion of 'name', system docs will need to have a section talking about how to add resources.